src/main/java/com/manardenza/utils/DatabaseSeeder.java

```diff
@@ -1,6 +1,70 @@
 package com.manardenza.utils;
 
+import com.manardenza.entity.Hotel;
+import com.manardenza.entity.Room;
+import com.manardenza.entity.User;
+import lombok.Getter;
+
+import java.util.ArrayList;
+import java.util.List;
+
 public class DatabaseSeeder {
 
-    // Class to fill DAO implementations
+    private static final String KIEV = "Kiev";
+    private static final String ODESSA = "Odessa";
+    private static final String LVOV = "Lvov";
+    private static final String DONETSK = "Donetsk";
+    private static final String ZAPOROZHYE = "Zaporozhye";
+    private static final String MARRIOTT = "Marriott";
+    private static final String HOLIDAY_INN = "Holiday Inn";
+    private static final String REIKARTZ = "Reikartz";
+
+    private DatabaseSeeder() {
+    }
+
+    @Getter private static List<User> usersList = new ArrayList<>();
+    @Getter private static List<Room> roomsList = new ArrayList<>();
+    @Getter private static List<Hotel> hotelsList = new ArrayList<>();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1213)